### PR TITLE
243 info to help copy

### PR DESCRIPTION
**Pull request by katjam**
Trello card: 
https://trello.com/c/EjALmoYN/243-copy-for-supporting-facts-actions-i-can-take-stage-ii

## Description

- Alter keys
- Add text
- Add keys to template page

@neontribe/sea-map

### CODE REVIEW

File: src/Page/HelpSelfSingle.elm [hunk 1]
```diff
@@ -99,6 +99,7 @@ renderResourceDetails resource =
         ++ [ verticalSpacing
            , p [] [ text (t resource.summary) ]
            , verticalSpacing
+           , text "Full guide: "
```
Comment (Nikomus) [inline src/Page/HelpSelfSingle.elm]: Should this be in a key?

Comment (katjam) [reply src/Page/HelpSelfSingle.elm]: Probably. I will move it.

File: src/Copy/Text.elm [hunk 2]
```diff
@@ -331,119 +356,172 @@ t key =
...
+
+        HelpSelfBankingResource2Link ->
+            "Opening a new bank account [PDF 469kB]"
+
+        -- Help Self single Debt
+        HelpSelfDebtTitle ->
+            "Information to support you with debt"
+
+        HelpSelfDebtResource1Title ->
+            "What is Coerced Debt?"
```
Comment (Nikomus) [inline src/Copy/Text.elm]: Only title with caps

File: src/Copy/Text.elm [hunk 3]
```diff
@@ -302,25 +321,31 @@ t key =
...
-        HelpSelfCategory5Link ->
-            "[cCc] Economic Abuse, Statistics and the Law"
+        HelpSelfCovidLink ->
+            "COVID-19 and my finances"
+
+        HelpSelfInfoLawLink ->
+            "Economic abuse and the law"
+
+        HelpSelfSeparatingLink ->
+            "Separating my finances from an abusive partner"
```
Comment (Nikomus) [inline src/Copy/Text.elm:345]: Is the mobile grid OK with this?

**Review by Nikomus — DISMISSED** [review]
A couple small Qs but looks fine to go in! 🌴

File: src/Copy/Text.elm (hunk 3 above)
Comment (katjam) [reply src/Copy/Text.elm:345]: Yep.

File: src/Copy/Text.elm (hunk 2 above)
Comment (katjam) [reply src/Copy/Text.elm]: Changed it!

**Review by Nikomus — APPROVED** [review]
All looks good to me! 🏤

**Pull request merged**